scripts/prepare-template.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+PACKAGE_LOCATION=$(pwd)/react-native-$1.tgz
+
+# Pack React Native into a `.tgz` file so we can run from source
+npm pack
+
+# Set the React Native version to point to the `.tgz` file
+node scripts/set-rn-template-version.js "file:$PACKAGE_LOCATION"
+
+# We need to generate CocoaPods project. To do so, we install depdendencies
+# locally and manually run `pod install`
+cd template
+
+npm install
+(cd ios && pod install)
```

Comment:
I think we'll want to pass `--repo-update` flag so we publish up-to-date pods

Reply:
Hmm... mind elaborating a bit what that flag is doing? We have local pods (from local spec within the repo, so no need to update the whole CocoaPods repo) and we also rm -rf Pods anyway.

Reply:
forgot it's local, you're right